motor_controller/motorbench-test-project/motorbench-test-project/rb_library/rb_isr.h

```diff
@@ -20,31 +20,31 @@ extern "C" {
 #include "rb_control.h"
 #include "rb_hall.h"
 #include "rb_pwm.h"
+#include "rb_board_ui.h"
  
 typedef enum 
 {   
     RBFSM_INIT = 0,
     RBFSM_STARTUP,
     RBFSM_RUNNING,
+    RBFSM_STOPPING,
     RBFSM_FAULTED
         
 } RB_FSM_STATE; 
 
-/** Global Variables */
+/** ISR Variables - These are passed into library functions as needed*/
 RB_MOTOR_DATA PMSM;
 RB_FSM_STATE state;
 RB_BOOTSTRAP bootstrap;
-
-bool bootstrapDone;
-
+RB_BOARD_UI boardUI;
+uint16_t speedLevel = 9;
 
```

Comment:
Doesn't have to be delt with rn but we should move the hall global variable to here for consistency